### PR TITLE
how to build servr with docker

### PR DESCRIPTION
**Pull request by mentik**
i already try to build docker image, but servr looks like not connected to the port on docker

### CODE REVIEW

**Comment by yihui** [comment]
I don't know much about Docker, and can't really help you here. Have you tried other port numbers?

**Comment by mentik** [comment]
yes, i am tried some port number still not listening the port, i also trying httpuv as comparison on docker, and its works

File: inst/examples/docker/app.R [hunk 1]
```diff
@@ -0,0 +1,7 @@
+require("servr")
+
+writeLines("\n\n\n *** Starting Servr *** \n\n\n")
+
+servr::create_server(port = 2222, handler = function(req) {
+    list(status = 200L, body = paste("Success:", req$PATH_INFO))
+  })
```
Comment (yihui) [inline inst/examples/docker/app.R:7]:
```suggestion
s = servr::create_server(port = 2222, host = '0.0.0.0', handler = function(req) {
    list(status = 200L, body = paste("Success:", req$PATH_INFO))
  })
print(s$url)
```

**Review by yihui — COMMENTED** [review]
That sounds odd. Try to print the URL and check if it is what you expected, e.g., whether it contains the expected port number. Another thing to try is `host = '0.0.0.0'`. I'm not sure if which `host` address you used in your **httpuv** example, but **servr** defaults to `127.0.0.1`.

File: inst/examples/docker/app.R (hunk 1 above)
Comment (mentik) [reply inst/examples/docker/app.R]: thank you its worked now

**Pull request closed without merging**